eZ/Publish/Core/Persistence/Legacy/Content/Type/Gateway.php

```diff
@@ -294,10 +294,10 @@
     /**
      * Returns field mapping data
      *
-     * Returns an associative array with ContentType and FieldDefinition identifiers as
-     * first and second level keys respectively, and FieldDefinition ID as value.
+     * Returns an associative array with ContentType and searchable FieldDefinition identifiers as
+     * first and second level keys respectively, and FieldType identifier as value.
      *
      * @return array
      */
-    abstract public function getFieldMap();
+    abstract public function getFieldMapData();
```

Comment:
No, it was intended for search only. I will change the name adding "searchable" both here and in handler interface/implementations.